charts/telepresence/values.yaml

```diff
@@ -110,3 +110,49 @@ licenseKey:
     #
     # Default: systema-license
     name: systema-license
+
+################################################################################
+## User Configuration
+################################################################################
+
+# Telepresence requires admin permissions for installation. Creating intercepts 
+# requires lower permissions. Use the following configurations to create roles 
+# so non-admin users can create intercepts.
+#
+# You do not need to create any extra RBAC if your users are cluster admins.
```

Comment:
Perhaps a nit (and also if you don't want to do this on the PR and just want me to add it to my next PR that's cool), but could we add a link to the docs about rbac: https://www.getambassador.io/docs/telepresence/latest/reference/rbac/ 
? That way they can go to the source of rbac in telepresence if they have questions + want the long explanation?